justfile

```diff
@@ -0,0 +1,57 @@
+ACT_VERSION := "0.2.32"
```

Comment:
we don't actually use just anywhere in this GitHub org, its just not standard tooling in the wider world/ecosystem, not in debian/ubuntu package repos, etc. can we switch this to a makefile.

Reply:
fair, I can swap this to a Makefile

Reply:
oh. although, we do have a few justfiles in the `cloud-custodian/cloud-custodian` repo in [tools/c7n_awscc/justfile](https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/c7n_awscc/justfile)

Reply:
yeah, that's a turd that should get removed its not really part of ci or dev either.